test/faucet.spec.ts

```diff
@@ -40,6 +42,37 @@ makeSuite('Faucet', (testEnv: TestEnv) => {
     it('Getter isPermissioned should return false', async () => {
       await expect(await faucetOwnable.isPermissioned()).is.equal(false);
     });
+
+    it('Mint function should mint tokens within limit', async () => {
+      const withinLimitThreshold = parseEther('100');
```

Comment:
better to fetch it from contract

Reply:
`5166eb05400757d705f83ffb61295fc2c85553d8`